lib/flagsmith/engine/evaluation/mappers.rb

```diff
@@ -0,0 +1,233 @@
+# frozen_string_literal: true
+
+module Flagsmith
+  module Engine
+    module EvaluationContext
+      module Mappers
+        # Using integer constant instead of -Float::INFINITY because the JSON serializer rejects infinity values
+        HIGHEST_PRIORITY = 0
```

Comment:
Just for safety, I'd prefer this to be lower than 0. Also, our naming convention for these constants is strongest-weakest:
```suggestion
        STRONGEST_PRIORITY = -99_999_999
```